shuup/core/models/_service_behavior.py

```diff
@@ -163,7 +164,7 @@ class GroupAvailabilityBehaviorComponent(ServiceBehaviorComponent):
     groups = models.ManyToManyField("ContactGroup", verbose_name=_("groups"))
 
     def get_unavailability_reasons(self, service, source):
-        if not source.customer.pk:
+        if not issubclass(type(source.customer), AnonymousContact) and not source.customer.pk:
```

Comment:
I really think that there is no reason to allow `request.customer` to be unsaved person or company contact.

But in case the order id done from admin the customer passed to behavior unavailability check might indeed be unsaved. Since the customer in admin order creation is saved only after confirm.